Description: Do not use relative path to mentos.py
This fails when testing against the installed package.
Author: Alexandre Ghiti <alexandre.ghiti@canonical.com>
Last-Update: 2022-01-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/test_pygments.rb
+++ b/test/test_pygments.rb
@@ -10,8 +10,13 @@
 class PygmentsHighlightTest < Test::Unit::TestCase
   RUBY_CODE = "#!/usr/bin/ruby\nputs 'foo'"
   RUBY_CODE_TRAILING_NEWLINE = "#!/usr/bin/ruby\nputs 'foo'\n"
+  begin
+    ruby_pygments_dir = Gem::Specification.find_by_name("pygments.rb").gem_dir
+  rescue Gem::MissingSpecError
+    ruby_pygments_dir = File.join(__dir__, '..')
+  end
   TEST_CODE = File.read(
-    File.join(__dir__, '..', 'lib', 'pygments', 'mentos.py')
+    File.join(ruby_pygments_dir, 'lib', 'pygments', 'mentos.py')
   )
 
   def test_highlight_defaults_to_html
